kube-client/src/lib.rs

```diff
@@ -194,6 +194,69 @@ mod test {
         Ok(())
     }
 
+    #[tokio::test]
+    #[ignore = "needs cluster (uses aggregated discovery, requires k8s 1.26+)"]
+    #[cfg(feature = "client")]
+    async fn aggregated_discovery_apis() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I actually don't see these tests running on CI. They should run in the coverage job but [that](https://github.com/kube-rs/kube/actions/runs/20561942635/job/59053775391?pr=1873)
outputs

```
 test config::incluster_config::test_kube_name ... ok
test test::aggregated_discovery_apis ... ignored, needs cluster (uses aggregated discovery, requires k8s 1.26+)
test test::aggregated_discovery_core ... ignored, needs cluster (uses aggregated discovery, requires k8s 1.26+)
```

it's possible the coverage job is getting truncated though (and it's actually there in another coverage pass). does it pass locally with `-- --ignored`?